userguide/content/en/docs/adding-content/shortcodes/index.md

```diff
@@ -194,14 +194,16 @@ This is placeholder content
 The **imgproc** shortcode finds an image in the current [Page Bundle](/docs/adding-content/content/#page-bundles) and scales it given a set of processing instructions.
 
 ```go-html-template
-{{</* imgproc spruce Fill "400x450" */>}}
-Norway Spruce <i>Picea abies</i> shoot with foliage buds.
-{{</* /imgproc */>}}
+{{%/* imgproc spruce Fill "400x450" */%}}
+Norway Spruce *Picea abies* shoot with foliage buds.
+{{%/* /imgproc */%}}
 ```
 
-{{< imgproc spruce Fill "400x450" >}}
-Norway Spruce <i>Picea abies</i> shoot with foliage buds.
-{{< /imgproc >}}
+Use the syntax printed above if the inner content and/or the `byline` parameter of your shortcode is authored in markdown. In case of HTML content, use square brackets `<>` as innermost delimiters: `{{</* imgproc >}}<b>HTML</b> content{{< /imgproc */>}}`.
```

Comment:
```suggestion
Use the syntax above if the inner content and/or the `byline` parameter of your shortcode is authored in markdown. In case of HTML content, use `<>` as innermost delimiters: `{{</* imgproc >}}<b>HTML</b> content{{< /imgproc */>}}`.
```